client/layout/guided-tours/tours/gdocs-integration-tour.js

```diff
@@ -0,0 +1,49 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+* Internal dependencies
+*/
+import {
+        makeTour,
+        Tour,
+        Step,
+        ButtonRow,
+        Quit,
+} from 'layout/guided-tours/config-elements';
+import {
+        isEnabled,
+        isSelectedSitePreviewable,
+} from 'state/ui/guided-tours/contexts';
+
+export const GDocsIntegrationTour = makeTour(
+        <Tour
+                name="gdocsIntegrationTour"
+                version="20160118"
+                path="/post"
+                when={ isEnabled( 'guided-tours/gdocs-integration-tour' ) }
+                >
```

Comment:
Minor, but FYI we've been indenting the trailing `>` at the same level as the opening `<Tour`; same goes for `Step` below.